### PR TITLE
Fix for cnos.py utility file for Python3.5

### PR DESCRIPTION
**Pull request by dkasberg**
##### ISSUE TYPE

 - Bugfix Pull Request

##### COMPONENT NAME

module_utils cnos.py utility file
##### ANSIBLE VERSION

```
ansible 2.3.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = Default w/o overrides
```

##### SUMMARY

fixes errors running with Python 3.5

```

```

### CODE REVIEW

**Comment by mattclay** [comment]
CI failure due to PEP 8 issue:

```
2017-02-22 02:18:23 ERROR: PEP 8: lib/ansible/module_utils/cnos.py:3171:17: E712 comparison to False should be 'if cond is False:' or 'if not cond:' (current)
```

PEP 8 tests can be run locally with `make pep8`.

File: lib/ansible/module_utils/cnos.py [hunk 1]
```diff
@@ -3168,7 +3168,7 @@ def waitForDeviceResponse(command, prompt, timeout, obj):
     obj.send(command)
     flag = False
     retVal = ""
-    while (flag is False):
+    while (flag == False):
```
Comment (abadger) [inline lib/ansible/module_utils/cnos.py]: This looks like it should be
``` python
while not flag:
```

**Comment by abadger** [comment]
Could you post the error that you're getting?  From looking at the changes, I'm afraid that this isn't actually fixing a Python3 error.  It's just avoiding the buggy code due to other implementation details when running on Python3.  If the implementation changes, the problem could resurface.

**Comment by dkasberg** [comment]
Hi abadger, thanks for the suggestion.  The developer has been out sick for the past day, but when he comes back I'll have him look at.

File: lib/ansible/module_utils/cnos.py [hunk 2]
```diff
@@ -3179,7 +3179,10 @@ def waitForDeviceResponse(command, prompt, timeout, obj):
             if(gotit != -1):
                 flag = True
         except:
-            retVal = retVal + "\n Error-101"
+            if(prompt != "(yes/no)?" ):
```
Comment (abadger) [inline lib/ansible/module_utils/cnos.py]: Note: conditionals should usually be written like this in python:
``` python
if prompt != "(yes/no)?":
```

leaving out the parenthesis as if and elif are not function calls.  The one time people break this general rule is when the conditional is long enough to span more than one line.  Then they write the conditional like this:
``` python
if (long_variable_name != "other_long_condtion_to_match"
    or second_variable in ('rest', 'of', 'condition')):
```
with a space between the if and the opening parenthesis.  (No need to change all of the existing code but a note to fix this up as you make changes).

**Comment by dkasberg** [comment]
Shippable stuck today?

**Comment by mattclay** [comment]
There was a major AWS outage today which prevented Shippable from running CI jobs. Things are starting to recover now, but there are quite a few jobs in the backlog.

**Pull request merged**